src/ForgeManager.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Laravel\Forge;
+
+use Illuminate\Support\Traits\ForwardsCalls;
+
+/**
+ * @mixin \Laravel\Forge\Forge
+ */
+class ForgeManager
+{
+    use ForwardsCalls;
+
+    /**
+     * The Forge instance.
+     *
+     * @var \Laravel\Forge\Forge
+     */
+    protected $forge;
+
+    /**
+     * Create a new Forge manager instance.
+     *
+     * @param  string  $token
+     */
+    public function __construct($token)
+    {
+        $this->forge = new Forge($token);
```

Comment:
What about forwarding the http client?

Reply:
@stevebauman we should address @siarheipashkevich's feedback, because the SDK doesn't expose 100% of the API and so it's highly likely you'll want to access the Http client here.

Reply:
Ok sounds good! How do you think we should grant access to the HTTP client?

We could just add a `guzzle()` method to the `Forge` instance to provide access?

```php
use Laravel\Forge\Facades\Forge;

Forge::guzzle()->get('...');
```

What do you guys think?

Reply:
Oh sorry I'm a dunce -- I get it, one moment!

Reply:
Done 👍